src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Editor/index.js

```diff
@@ -63,6 +64,8 @@ export default function ServiceJourneyEditor(props) {
 
   const isBlankName = isBlank(name);
 
+  const ToolTipIfError = !validPassingTimes ? Tooltip : Fragment;
```

Comment:
Vad gör `Fragment`?

Reply:
Ingenting, er for å vise Tooltip kun om det er feil